### DISCUSSION TITLE
Why windows binary require VCRUNTIME140.dll?

### DISCUSSION THREAD

**Discussion by qarmin** [slint-ui/slint#3969] [Q&A] [answered]
Recently I compiled my app to check if it will work fine on windows, but I found that I cannot run binary because it needs VCRUNTIME140.dll.

Why this is needed, when I'm using only rust in app without any external dependencies?

Binaries - https://github.com/qarmin/czkawka/actions/runs/6933877290 - krokiet at the bottom of page

**Comment by Vadoola** [comment, marked as answer]
I could be mistaken but I believe that DLL is needed for any rust program compiled using the MSVC toolchain instead of the Gnu toolchaim on Windows.
